README.md

```diff
@@ -1,22 +1,27 @@
-Elasticsearch For Beginners: Generate and Upload Randomized Test Data
-====
+# Elasticsearch For Beginners: Generate and Upload Randomized Test Data
+
 Because everybody loves test data.
 
+## Ok, so what is this thing doing?
 
-### Ok, so what is this thing doing?
+`es_test_data.py` lets you generate and upload randomized test data to
+your ES cluster so you can start running queries, see what performance
+is like, and verify your cluster is able to handle the load.
 
-`es_test_data.py` lets you generate and upload randomized test data to your ES cluster so you can start running queries, see what performance is like, and verify your cluster is able to handle the load.<br>
-It allows for easy configuring of what the test documents look like, what kind of data types they include and what the field names are called.
+It allows for easy configuring of what the test documents look like, what
+kind of data types they include and what the field names are called.
 
 ## Cool, how do I use this? 
 
-#### Prerequisites
-
-Let's assume you have an Elasticsearch cluster running. If not, [set it up locally](http://ohardt.us/es-install) and point your browser to [http://localhost:9200](http://localhost:9200) to see if it's up.
+Let's assume you have an Elasticsearch cluster running. If not,
+[set it up locally](http://ohardt.us/es-install) and point your browser to
+[http://localhost:9200](http://localhost:9200) to see if it's up.
 
-Python and [Tornado](https://github.com/tornadoweb/tornado/) and [NumPy](http://www.numpy.org/) are used, run `pip install tornado numpy` to install Tornado and NumPy if you don't have it already.
+Python and [Tornado](https://github.com/tornadoweb/tornado/) and
+[NumPy](http://www.numpy.org/) are used, run `pip install tornado numpy` to
```

Comment:
The numpy dependency was removed in https://github.com/oliver006/elasticsearch-test-data/commit/4d31f6742e62cd992b7fb30612894abd640495f9 - you can remove all references to it from the README.